device/sycl/src/clusterization/component_connection.hpp

```diff
@@ -24,8 +23,9 @@ namespace traccc::sycl {
 void component_connection(
     cluster_container_view clusters_view,
     const cell_container_types::host& cells_per_event,
-    vecmem::data::vector_view<unsigned int> clusters_count_view,
-    vecmem::unique_alloc_ptr<unsigned int>& total_clusters,
-    vecmem::memory_resource& resource, queue_wrapper queue);
+    vecmem::data::jagged_vector_view<unsigned int> sparse_ccl_indices_view,
+    vecmem::data::vector_view<std::size_t> cluster_prefix_sum_view,
+    const std::size_t& cells_max, vecmem::memory_resource& resource,
```

Comment:
Just pass `cells_max` by value. Primitive types should in 99% of the cases just be passed by value.

Reply:
Done.